tests/expectations/core/test_expect_column_values_to_be_string_integers_increasing.py

```diff
@@ -0,0 +1,388 @@
+import pandas as pd
+import pytest
+
+from contrib.experimental.great_expectations_experimental.expectations.expect_column_values_to_be_string_integers_increasing import (
+    ExpectColumnValuesToBeStringIntegersIncreasing,
+)
+from contrib.experimental.great_expectations_experimental.metrics.column_values_string_integers_increasing import (
+    ColumnValuesStringIntegersIncreasing,
+)
+from great_expectations.core.batch import RuntimeBatchRequest
+from great_expectations.core.expectation_validation_result import (
+    ExpectationValidationResult,
+)
+from great_expectations.data_context import DataContext
+from great_expectations.exceptions.exceptions import MetricResolutionError
+from great_expectations.self_check.util import (
+    build_sa_validator_with_data,
```

Comment:
This can be removed since SQL isn't implemented.

Reply:
Resolved by moving tests to contrib.